superset-frontend/src/explore/components/ControlPanelsContainer.tsx

```diff
@@ -58,6 +60,8 @@ import { ChartState } from 'src/explore/types';
 import ControlRow from './ControlRow';
 import Control from './Control';
 import { ControlPanelAlert } from './ControlPanelAlert';
+import { RunQueryButton } from './RunQueryButton';
+import { Tooltip } from '../../components/Tooltip';
```

Comment:
Don't forget to change the order of the imports 😉 

```suggestion
import { Tooltip } from 'src/components/Tooltip';
```